### ISSUE TITLE
Missing Pieces

### ISSUE THREAD

**Issue by jaxels20** [rnygar20/P7#15]
## METHOD

- [x] hyperparameters gat
- [x] hyperparameters graphsage
- [x] hyperparameters gcn
- [x] intro gat
- [x] intro graphsage
- [x] specify the datasplit. (cite paper from pytorch)
- [x] specify how weights are calculated 
- [ ] software
- [x] change label dist plot to fit our theme "blue" ( VI KAN LIGE SNAKKE OM DEN HER)

## RESULTS 

- [x] Write something for the table/figures 
- [x] Note (remember to write that the significance can not be noted on the table for average and if it can do it)
- [x] signify significance on the plot for subfigures (in the x axis values)
- [x] fix error in table 

## DISCUSSION 
- [x] Interpretation of Model Performance
- [x] Effectiveness in Handling Class Imbalance
- [x] Computational Efficiency
- [x] figure of computational efficiency
- [x] Robustness
- [x] Limitations

## FURTHER WORK 
- [ ] Where to place it in the report 
- [ ] write it and come up with ideas 
- [ ] just very short

## GENERAL
- [ ] References are missing everywhere:) 
- [ ]  Conclusion
- [ ] abstract 
- [ ] related work
